src/applications/vaos/services/appointment/transformers.js

```diff
@@ -119,10 +122,7 @@ function getRequestStatus(request, isExpressCare) {
  * @returns {String} Appointment status
  */
 function getConfirmedStatus(appointment, isPast) {
-  const currentStatus =
-    getAppointmentType(appointment) === APPOINTMENT_TYPES.ccAppointment
```

Comment:
This is unnecessary now that we're checking for vds or vvs appointments directly in getVistaStatus